nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -451,6 +451,10 @@ def __init__(
         # will be set in self.forward() if defined in AccessMixin config
         self.interctc_capture_at_layers = None
 
+    def set_dropout(self, dropout):
```

Comment:
add doc strings. Is this part of original `SimIPL` algorithm?

Reply:
Added doc strings.
Yes.

Reply:
Were there tests on the affects of this per layer dropout? It seems a tad unnecessary given the additional dropout in the rest of the conformer.

Reply:
I added this to be able to change dropout in the middle of the training(as in SlimIPL paper), but algorithm works better when we have the same dropout through the whole training.
So I believe we can remove this.